app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+
+const { getTopics } = require("./controllers/controllers.topics");
+const { handle500s, handle404s } = require("./controllers/errors");
+
+app.get("/api/topics", getTopics);
+
+app.all("*", handle404s);
+
+app.use(handle500s, (req, res, next) => {
+  next();
```

Comment:
a few things here, the 500 error handler is last in the chain so you don't need to invoke `next` here;
express will only see this as an error handler if you provide 4 args e.g. `err, req, res, next`;
in this error handler you want to be both console logging the `err` and sending a response with `500`

Reply:
in the app.use or in the actual error function? thanks for the feedback